src/rules/team/parent.py

```diff
@@ -3,4 +3,5 @@
 
 def check_parent_link(issue: jira.resources.Issue, context: dict, _: bool) -> None:
     if issue.raw["Context"]["Related Issues"]["Parent"] is None:
+        context["non-compliant"] = True
```

Comment:
Should we also set the issue as non-compliant when the end date is unset in `src/rules/team/target_dates.py`? How about the start date, is it critical enough to add the label?

I don't know if that's necessary since your bot will send an email, but it does require manual action. Do you track the health of the project? If so it, might make it easier to get stats on the number of stories that are not compliant.

Reply:
It's a good question. Let's leave it out of scope here but keep it in mind.

I was thinking that the "flagged" field in JIRA might be nice to start using here instead of a label. It highlights the issue when viewing it on a kanban or scrum board. It could also be nice to set it when it is detected that the "target end" date is _after_ the "due date". Those would be nice to see easily; however, flagging and putting alarms on the issues might send out signals that I'm not 100% sure I want to send. I don't want to create a situation where people feel like they need to pull in their target end date just to appease the automation.